### ISSUE TITLE
Bootstrap options missing from help

### ISSUE THREAD

**Issue by stianst** [keycloak/keycloak#32176]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

dist/quarkus

### Describe the bug

The bootstrap options are not shown in help. This applies to CLI help (`kc.sh start|start-dev --help`) as well as in the `All configuration guide`.

This includes at least the following options:

* `bootstrap-admin-username`
* `bootstrap-admin-password`
* `bootstrap-admin-client-id`
* `bootstrap-admin-client-secret`

This makes it hard to use these options as the only place to discover them in the `Admin bootstrap and recovery` guide. As I can't seem to get these options quite right, or forget them, I need to then open the guide instead of just looking at the help for reference.

### Version

main

### Regression

- [ ] The issue is a regression

### Expected behavior

`All configuration` guide has a `Bootstrap` section, which lists the options. 

`kc.sh start|start-dev --help` lists a `Bootstrap` section, which lists the options.

### Actual behavior

The bootstrap options are hidden from both the `All configuration` guide and CLI help

### How to Reproduce?

* Run `./kc.sh start-dev --help | grep bootstrap` 
* Observe no options are listed

**Comment by shawkins** [comment]
@stianst hiding them initially was intentionally due to their narrow usage. Can others in @keycloak/cloud-native can weigh in whether it makes sense to generally expose these?

**Comment by mabartos** [comment]
@shawkins Makes sense to me to show them as those are also included in the Admin bootstrap guide. I don't see any problem to show them by default.

**Comment by vmuzikar** [comment]
@shawkins As I mentioned in a private discussion, IIRC the original reasoning to hide them was that they should've been primarily used as env vars. However, this has been evolved and we now support (and encourage) usage of the CLI args as well. For me it makes sense to expose them.

**Comment by shawkins** [comment]
> we now support (and encourage) usage of the CLI args as well. For me it makes sense to expose them.

That could be true for the client id and the username, but the password and client secret would not be encouraged for use via the CLI. How involved should the option descriptions be? Should they mention preferring env variables for the secret stuff and that they are only applicable when the master realm is created?

**Comment by vmuzikar** [comment]
> Should they mention preferring env variables for the secret stuff and that they are only applicable when the master realm is created?

From my perspective, yes and yes. :)
